spec/lib/web_of_science/queries_spec.rb

```diff
@@ -140,6 +142,37 @@
     end
   end
 
+  shared_examples 'search_params' do
+    let(:query) { params[:queryParameters] }
```

Comment:
no need for these `let` statements here.  In the examples below, just do:
```ruby
expect(params[:queryParameters]).to include ...
expect(params[: retrieveParameters]).to include ...
```
Or if you want to be paranoid/fancy, you can combine all 3 of these into one expectation:
```ruby
expect(params).to match a_hash_including(
  query_parameters: a_hash_including(databaseId: String, userQuery: String, timeSpan: Hash, queryLanguage: String),
  retrieveParameters: a_hash_including(firstRecord: Integer, count: Integer, option: Array)
)
```
I understand if you think that the rspec output might be more intimidating, but the point is that the let statements aren't giving us anything.

Reply:
This habit of extracting a field helps to keep line lengths shorter and that keeps rubocop happy.  As this PR is already approved and this is a style issue, I'm going to take this advice for future work but skip it here in this PR (I want to move forward with more work on harvesting and want to avoid additional git conflicts or anything that might come up from changing this).